bindings/wasm/src/document.rs

```diff
@@ -118,6 +124,17 @@ impl Document {
     self.0.remove_method(&did.0).map_err(err)
   }
 
+  #[wasm_bindgen(js_name = insertService)]
+  pub fn insert_service(&mut self, service: Service) -> Result<bool, JsValue> {
```

Comment:
```suggestion
  pub fn insert_service(&mut self, service: &Service) -> Result<bool, JsValue> {
```

and just `clone` what you need internally, otherwise this will consume the `Service` and reusing it on the JS side will throw an error